.github/workflows/codeql-analysis.yml

```diff
@@ -13,12 +13,6 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ "main" ]
```

Comment:
should we fix the ubuntu version here?

Reply:
I think it was already discussed in one of the PRs. Afair the conclusion was that it's irrelevant for the results of this analysis.
But I'm fine with fixing the version if you want.

Reply:
Nope, just double checking. Don't worry about it :)